CMakeLists.txt

```diff
@@ -564,10 +547,10 @@ if (NANOGUI_BUILD_EXAMPLES)
   add_executable(example4      src/example4.cpp)
   add_executable(example_icons src/example_icons.cpp)
 
-  target_link_libraries(example1      nanogui)
+  target_link_libraries(example1      nanogui glfw)
```

Comment:
This is odd as well, though without it I get `error adding symbols: DSO missing from command line`. It's something to do with nanogui shared vs static, this error doesn't occur if `NANOGUI_BUILD_SHARED=0` is specified.